pandas/tests/indexing/test_floats.py

```diff
@@ -507,30 +507,22 @@ def test_floating_misc(self):
         # label based slicing
         result1 = s[1.0:3.0]
         result2 = s.loc[1.0:3.0]
-        result3 = s.loc[1.0:3.0]
```

Comment:
not necessarily for this PR, but i think this test can be parametrized over indexer_sl

Reply:
Yes, good point. Was too focused on the duplicates :)